debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/processor/AbstractLogMinerEventProcessor.java

```diff
@@ -535,10 +555,18 @@ protected void handleSchemaChange(LogMinerEventRow row) throws InterruptedExcept
                             row.getRedoSql(),
                             getSchema(),
                             row.getChangeTime(),
-                            metrics));
+                            metrics,
+                            event -> processTrucateEvent(row, event)));
         }
     }
 
+    private void processTrucateEvent(LogMinerEventRow row, DdlParserListener.TableTruncatedEvent event) {
```

Comment:
```suggestion
    private void processTrucateEvent(LogMinerEventRow row) {
```

Is there a reason why we need to pass something from the ANTLR parser directly into this code path?

Reply:
no, can be removed indeed